internal/pkg/skuba/addons/cilium.go

```diff
@@ -27,11 +27,7 @@ import (
 )
 
 func init() {
-	registerAddon(kubernetes.Cilium, renderCiliumTemplate, ciliumCallbacks{}, normalPriority, []getImageCallback{GetCiliumInitImage, GetCiliumOperatorImage, GetCiliumImage})
-}
-
-func GetCiliumInitImage(imageTag string) string {
-	return images.GetGenericImage(skubaconstants.ImageRepository, "cilium-init", imageTag)
+	registerAddon(kubernetes.Cilium, renderCiliumTemplate, ciliumCallbacks{}, normalPriority, []getImageCallback{GetCiliumOperatorImage, GetCiliumImage})
```

Comment:
Why do we change this remove if we still use a Init image? I am confused, and the commit message is a bit terse. I see/suppose that the init image is now the same as the cilium image, but I think it would have been clearer to be explicit in a commit message. Though, it's the morning, so I might have missed it :laughing: .

Reply:
We don't have an init image anymore. And yes, we use the same image for init container and the main cilium container now. I will mention that in the commit message.